### ISSUE TITLE
function-name-case

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1052]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Function name can be in any-case

**Comment by jeddy3** [comment]
Sgtm.

```css
/* function-name-case: "upper" */
a { margin: CALC(5% - 10em); }
```

- Name: `function-name-case`
- Primary options: `"upper"|"lower"`
- Secondary options: none
- Message: `Expected "${actual}" to be "${expected}" (function-name-case)`.

**Comment by evilebottnawi** [comment]
@jeddy3 WIP

**Comment by jeddy3** [comment]
@davidtheclark Can I add @evilebottnawi as a contributor please? If only so they can label the "wip" status of things :)

**Comment by davidtheclark** [comment]
I just did yesterday, right @evilebottnawi ?

**Comment by evilebottnawi** [comment]
@davidtheclark cool :smile: I just did not see.

**Comment by evilebottnawi** [comment]
Merged. Close.

**Comment by jeddy3** [comment]
Welcome aboard @evilebottnawi :)
